lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -45,6 +49,32 @@ def process_root_folder
                 end
             end
 
+            # Transfer logs from a process server to the main computer server
+            #
+            # @param [Pathname] src_dir the log folder on the process server
+            # @param [Pathname] dst_dir the folder on the main computer
+            #   in which to transfer the logs to
+            def process_transfer(src_dir, dst_dir)
+                return unless (mng = log_transfer_manager)
+
+                candidates = self.class.find_all_dataset_folders(@src_dir)
+                candidates.each do |child|
+                    mng.transfer(child)
+                end
+            end
+
+            def log_transfer_prepare
+                return unless Syskit.conf.log_transfer.ip
+
+                conf = Syskit.conf.log_transfer
+                conf.target_dir ||= log_dir
+                @log_transfer_manager = LogTransferManager.new(conf)
```

Comment:
You won't have access to these configurations within log_runtime_archive. It does not load a robot configuration. Need to pass all arguments as command line.